src/main.rs

```diff
@@ -20,9 +25,97 @@ enum Opt {
         /// Verbose mode (-v, -vv, -vvv, etc.)
         #[clap(short, long, parse(from_occurrences))]
         verbose: u8,
+
+        #[clap(short, long)]
+        google_sheet: Option<String>,
     },
 }
 
+struct Collection<T: google_sheets::IntoSheetEntry> {
+    headers: Vec<String>,
+    data: Vec<T>,
+    num_columns: usize,
```

Comment:
Can't we get this from `headers.len()`?

Reply:
Yes, which we do. This is unused. And we get `num_rows` from `data.len()`

I will remove these.